sql/hive/pom.xml

```diff
@@ -133,7 +133,6 @@
                 </goals>
                 <configuration>
                   <sources>
-                    <source>src/test/scala</source>
```

Comment:
I think I've been bitten by this before; we may have to pass an option to ensure overriding.